queries.sql

```diff
@@ -79,3 +79,20 @@ SELECT MIN(weight_kg) FROM animals;
 
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth  BETWEEN '1990-01-01' AND '2000-12-31';
+
+
+-- Write queries (using JOIN) to answer the following questions
+
+SELECT names, full_name FROM animals INNER JOIN owners ON animals.owners_id = owners.id WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.names, species.names FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.names = 'Pokemon';
+
+SELECT owners.full_name, animals.names FROM owners LEFT JOIN animals ON owners.id = animals.owners_id;
+
+SELECT COUNT(animals.names), species.names FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.names;
+
+SELECT owners.full_name, animals.names, species.names FROM owners INNER JOIN animals ON owners.id = animals.owners_id INNER JOIN species ON species.id = animals.species_id WHERE species.names = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
```

Comment:
- [ ] This query is wrong. You are supposed to select from animals table and join the species table. Kindly ake this change.

- [ ] Kindly include its screenshot.

![image](https://user-images.githubusercontent.com/86318284/207672144-73414671-8004-4309-99db-589883ef9f63.png)

Reply:
Thank you for your time.
![orwell](https://user-images.githubusercontent.com/105460631/207729153-87d05e54-b3ad-4de2-9f52-c3877e68d04c.png)